draft-ietf-privacypass-architecture.md

```diff
@@ -436,203 +415,178 @@ clients cannot determine if this value is correct or otherwise a tracking
-policy" that documents the procedures that the server uses to ensure
-that client privacy is respected. If a server does not publish such a
-document then the client may choose to use its own judgement, or to
-reject the server altogether.
-
-It should be noted that the client trust decision can be made apriori by
-specifying an allow-list of all servers that it accepts tokens from.
-This means that these checks do not have to be performed online.
+## Issuer key rotation
 
-# Privacy considerations {#privacy}
-
-In the Privacy Pass protocol {{?I-D.ietf-privacypass-protocol}}, redemption
-tokens intentionally encode very little information beyond which key was
-used to sign them. The protocol intentionally uses components that
-provide cryptographic guarantees of this fact. However, even with these
-guarantees, the way that the ecosystem is constructed can be used to
-identify clients based on this limited information.
-
-The goal of the Privacy Pass ecosystem is to construct an environment
-that can easily measure (and maximize) the relative anonymity of any
-client that is part of it. An inherent feature of being part of this
-ecosystem is that any client can only remain private relative to the
-entire space of users using the protocol. Moreover, by owning tokens for
-a given set of keys, the client's anonymity set shrinks to the total
-number of clients controlling tokens for the same keys.
-
-In the following, we consider the possible ways that servers and servers
-can leverage their position to try and reduce the anonymity sets that
-clients belong to (or, user segregation). For each case, we provide
-mitigations that the Privacy Pass ecosystem must implement to prevent
-these actions.
-
-## Server key rotation
-
-Techniques to introduce client "segregation" can be used to reduce
-client anonymity. Such techniques are closely linked to the type of key
-schedule that is used by the server. When a server rotates their key,
-any client that invokes the issuance protocol in this key cycle will be
+Techniques to introduce Client "segregation" can be used to reduce
+Client anonymity. Such techniques are closely linked to the type of key
+schedule that is used by the Issuer. When an Issuer rotates their key,
+any Client that invokes the issuance protocol in this key cycle will be
 part of a group of possible clients owning valid tokens for this key. To
-mechanize this attack strategy, a server could introduce a key rotation
-policy that forces clients into small key cycles. Thus, reducing the
-size of the anonymity set for these clients.
+mechanize this attack strategy, an Issuer could introduce a key rotation
+policy that forces Clients into small key cycles. Thus, reducing the
+size of the anonymity set for these Clients.
 
-We RECOMMEND that servers should only invoke key rotation for fairly
+It is RECOMMENDED that Issuers should only invoke key rotation for fairly
 large periods of time such as between 1 and 12 weeks. Key rotations
```

Comment:
```suggestion
Issuers SHOULD invoke key rotation for
a period of time between 1 and 12 weeks. Key rotations
```
RECOMMENDED and SHOULD have the same semantic, and SHOULD seems more direct here. Also, if there's a SHOULD, the recommendation should be clear.